data/charts/crds/templates/crds.yaml

```diff
@@ -1,7 +1,9 @@
 {{ .Files.Get "files/crd-10.yaml" }}
 {{ .Files.Get "files/crd-11.yaml" }}
 {{ .Files.Get "files/crd-12.yaml" }}
+{{- if .Values.certmanager }}
```

Comment:
should this change not go in upstream/istio-installer?  I am very concerned we are creating 3 different sets of gotpl instead of 2 looking at this pr.

Reply:
yes, but we're not ready to do that yet, because if something needs to be fixed in installer i don't want to block progress in operator on that. right now there are lots of such fixes.
for the time being i'm just syncing these manually but once installer e2e tests are running with the operator (this week i hope) the plan is to do a final sync and delete this copy of the charts.